tv_apps/smart-home/js/home.js

```diff
@@ -165,6 +165,46 @@
       }
     },
 
+    _fillDeckIcon: function(cardButton, card) {
+
+      // deck's icon using gaia font
+      var deckIcon = document.createElement('span');
+      deckIcon.className = 'icon';
+      deckIcon.dataset.icon = card.fontIcon;
+
+      // front wave of a deck
+      var waveFront = document.createElement('div');
```

Comment:
How about create a method call "createWave"?
The lines creating wave are totally the same but different in class names.